humanize-duration.js

```diff
@@ -1511,7 +1512,8 @@
   /**
    * Create a humanizer, which lets you change the default options.
    *
-   * @param {Options} [passedOptions]
+   * @param {Options} [passedOptions] Options to customize the humanizer
+   * @returns {function(number, Options): string} A function that humanizes durations
```

Comment:
Could you change this to a TypeScript-style type? Something like `(ms: number, humanizerOptions: Options) => string`.